src/Pages/Login/Signup.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import './Signup.scss';
+
+class Signup extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      password: '',
+      passwordReCheck: '',
+      name: '',
+      phone_number: '',
+    };
+  }
+
+  fetch = e => {
+    e.preventDefault();
+    fetch('http://10.58.2.175:9000/users/signup', {
```

Comment:
```suggestion
    e.preventDefault();
    
    fetch('http://10.58.2.175:9000/users/signup', {
```
구분되는 로직들은 한줄 씩 띄워주세요!